metadata_harvester_cli.py

```diff
@@ -15,12 +17,15 @@
     )
 def retrieve_metadata_content(url):
     """
-    Fetch all metadata records.
+    Fetch metadata records and transform them to JSON.
     """
     api = PMH_API(url)
-    metadata_contents = api.get_all_metadata_records()
+    metadata_contents = api.get_all_metadata_records(limit=3) #limit to only three for initial testing purposes
```

Comment:
For the scope of this ticket, is the code supposed to work for the whole data? Because when I remove the limit, I'm getting issues for some of the records, namely the `_get_text_xpath()` function returns an empty list sometimes, at least for some persistent identifiers. 
If the nature of the data is so that sometimes these elements are empty and it's OK, then these cases should probably be handled in the code. If the elements should never be empty, there might be some bug in the code.

Reply:
As discussed, there are deleted records that do not have an identifier anymore and the code fails. Will fix now that we know how to deal with them.

Reply:
Will the issue of deleted records be solved in a future ticket? I'm wondering if this issue is resolved even though the harvesting still fails without a limit of 3.

Reply:
The issue of deleted records will be resolved in a future ticket.